### PR TITLE
Do a sample Code review

### PR DESCRIPTION
**Pull request by sabetech**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,28 +1,101 @@
-<!DOCTYPE html>
-<html lang="en">
+<html>
+    <head>
+        <link href="css/style.css" rel="stylesheet" >
```
Comment (sabetech) [inline index.html:9]: Meta tags should go here. They help give more description to your site

File: index.html [hunk 2]
```diff
@@ -1,28 +1,101 @@
...
+        <figure id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" />
+            <figcaption id="img-caption">Dr. Norman Borlaug, third from left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger</figcaption>
+        </figure>
 
-    For now, the test suite only works in Chrome!
-    Please read the README below in the JS Editor before beginning.
-    Feel free to delete this message once you have read it.
-    Good luck and Happy Coding!
+        <div class="history" id="tribute-info">
```
Comment (sabetech) [inline index.html]: Instead of a div, you could a section here.

**Review by sabetech — COMMENTED** [review]
Looks like the you pasted the html code twice. Check and make that fix

File: index.html (hunk 1 above)
Comment (Wes-Isaac) [reply index.html:9]: - [x] t's better to be more polite you can say _Kindly use this..._ _or it's better to..._

File: index.html (hunk 2 above)
Comment (Wes-Isaac) [reply index.html]: - [x] Good job phrasing your comment. but you have to also provide a reason why you are requesting this particular change and also it's better to be more polite you can say _Kindly use this..._

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @sabetech , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (sabetech) [reply index.html:9]: Thanks for the feedback

File: index.html (hunk 2 above)
Comment (sabetech) [reply index.html]: Feedback appreciated

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @sabetech , 

Great work so far 👏 
But you have not applied the requested changes. You are to edit your review comments following the previous reviewer's suggestions. So just go under his comments and add your edited review comments and you will be good to go 👍 


 ## Required Changes ♻️
_Check the comments under your review comments._
- [ ] (OPTIONAL) It would be great if you could add checkboxes to your comments so that the student can tick them when the issue has been resolved 💯 
- [ ] (OPTIONAL) It is nice to add emojis to your suggestions to make them more friendly 😄 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Kingstalux) [reply index.html:9]: Add your updated comments here.

File: index.html (hunk 2 above)
Comment (Kingstalux) [reply index.html]: Add your updated comments here.

File: index.html (hunk 1 above)
Comment (sabetech) [reply index.html:9]: Kindly use Meta tags in the head section of your website. They help to give more description to your site

File: index.html (hunk 2 above)
Comment (sabetech) [reply index.html]: Kindly use the section tag instead of div. This is semantically more accurate than using a div.

**Review by Wes-Isaac — APPROVED** [review]
Hi @sabetech , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**